libraries/AP_HAL_ChibiOS/hwdef/ORBITH743/defaults.parm

```diff
@@ -0,0 +1,8 @@
+# setup for LEDs on chan13
+SERVO13_FUNCTION 120
+
+# USART1 for ESC Telemetry
```

Comment:
these are dupilcates of those  in hwdef (the proper place to set UART  default protocols)